packages/api-fetch/README.md

```diff
@@ -0,0 +1,42 @@
+# @wordpress/api-fetch
+
+Wrapper around `window.fetch` to call WordPress REST APIs.
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/api-fetch --save
+```
+
+## Usage
+
+```js
+import apiFetch from '@wordpress/api-fetch';
+
+apiFetch( { path: '/wp/v2/posts' } ).then( posts => {
+	console.log( posts );
+} );
+```
+
+### Middlewares
+
+the `api-fetch` package supports middlewares. Middlewares are functions you can use to wrap the `wp.apiFetch` calls to perform any pre/post process to the API requests.
+
+**Example**
+
+```js
+wp.apiFetch.use( ( options, next ) => {
+	const start = Date.now();
+	const result = next( options );
+	result.then( () => {
+		console.log( 'The request took ' + Date.now() - start );
+	} );
+	return result;
+} );
+```
+
+The `api-fetch` package provides built-in middlewares you can use to provide a `nonce` and a custom `rootURL`.
```

Comment:
Should the `preloading` and `namespace-endpoint` middlewares be mentioned here too?

Reply:
Also, how does one leverage said nonce and rootURL middlewares ?